app/main.py

```diff
@@ -16,4 +16,14 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mutable_variables = [
+    my_favourite_films, marks, collection_of_coins
+]
+immutable_variables = [
+    lucky_number, pi, one_is_a_prime_number, name, profile_info
+]
+
+sorted_variables = {
+    "mutable": mutable_variables, "immutable": immutable_variables
+}
+print(sorted_variables)
```

Comment:
print is useless heere

Reply:
@vasylhnatiuk should be good now:
https://github.com/mate-academy/py-mutable-immutable/pull/1143/files
Just forgot to rename new commit